pkg/controllermanager/apis/config/v1alpha1/defaults_test.go

```diff
@@ -17,32 +17,36 @@ package v1alpha1_test
 import (
 	"time"
 
+	configv1alpha1 "github.com/gardener/gardener/pkg/controllermanager/apis/config/v1alpha1"
```

Comment:
Nit: Usually, we dot-imported the tested package 
```suggestion
	. "github.com/gardener/gardener/pkg/controllermanager/apis/config/v1alpha1"
```